operator/beiboot/utils.py

```diff
@@ -224,3 +148,76 @@ def get_taken_gefyra_ports(
 def get_namespace_name(name: str, cluster_config: ClusterConfiguration) -> str:
     namespace = f"{cluster_config.namespacePrefix}-{name}"
     return namespace
+
+
+class AsyncTransition(Transition):
```

Comment:
this part is hard to understand for me, maybe because I never worked with the statemachine package.